mlir/lib/Dialect/OpenMP/IR/OpenMPDialect.cpp

```diff
@@ -2432,6 +2432,22 @@ LogicalResult DistributeOp::verifyRegions() {
   return success();
 }
 
+//===----------------------------------------------------------------------===//
+// DeclareMapperOp / DeclareMapperInfoOp
+//===----------------------------------------------------------------------===//
+
+LogicalResult DeclareMapperInfoOp::verify() {
+  return verifyMapClause(*this, getMapVars());
+}
+
+LogicalResult DeclareMapperOp::verifyRegions() {
+  if (!isa<DeclareMapperInfoOp>(
```

Comment:
Nit: We wouldn't want to crash the verifier if the block was empty.
```suggestion
  if (!isa_and_present<DeclareMapperInfoOp>(
```